utils/logger.c

```diff
@@ -0,0 +1,6 @@
+#include "logger.h"
+static int TA_LOG_LEVEL = LV_INFO;
```

Comment:
You should implement non-trivial logger, which takes circular buffer into consideration.